mbed_host_tests/host_tests/base_host_test.py

```diff
@@ -64,6 +64,14 @@ def reset_dut(self, value):
         if self.__event_queue:
             self.__event_queue.put(('__reset_dut', value, time()))
 
+    def reset(self):
```

Comment:
I know I wrote this code pretty hastily, but let's get the keys matching the function names if at all possible.

Can we make this function `soft_reset_dut`?

This would mean you'd need to update your custom host test as well.